lib/Makefile.am

```diff
@@ -29,7 +30,7 @@ libmptcpd_la_SOURCES =		\
 	hash_sockaddr.c		\
 	hash_sockaddr.h
 
-#if BUILDING_DLL
+#IF BUILDING_DLL
```

Comment:
Is the all-caps `#IF` here accidental?

Reply:
Yes.  Good catch.  That Automake conditional is commented out, too.  Any objections to removing it entirely?

Reply:
No objection to removing it